README.md

```diff
@@ -6,6 +6,11 @@ This repo contains the source code of https://howmuch.green
 
 Node 16 is required.
 
+>  💡 On a Mac M1 you'll need to execute the following command prior to `npm install`:
```

Comment:
Could we add a link to the canvas PR thread? So we know where that comes from + it may change at some point

Reply:
done